src/pages/sorting/Cards.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import { FaAngleRight } from "react-icons/fa6";
+import "./Cards.css";
```

Comment:
please use module style for importing css.

Can we also separate each component to a folder. Like
```
Cards
   -> Cards.jsx
   -> cards.module.css
SortingPage
    -> SortingPage.jsx
    -> sortingPage.module.css
```

Reply:
Also please use `.view.jsx` for this file to make it Cards.view.jsx`